WordPressVIPMinimum/Sniffs/Functions/RestrictedFunctionsSniff.php

```diff
@@ -311,6 +311,13 @@ public function getGroups() {
 					'get_children',
 				],
 			],
+			'deprecated' => [
```

Comment:
This is tipping into PHPCompatibility territory (as was the previous sniff).

However, thinking ahead, if function `x()` got deprecated in PHP 7.3, then we'd either have to generalise the message to not contain the version, or add in another `deprecated` (but with a different name) category.

There are lots of [deprecations in PHP 7.2](https://secure.php.net/manual/en/migration72.deprecated.php) - so why does this one get special treatment?

Reply:
I think it would be safe to take it out and leave the PHP deprecation notices to PHPCompatibility.

Reply:
Fair point @rebeccahum.

@tessaneedham Do you know why `create_function` gets attention in the Docs, yet none of the other deprecations for 7.2 get mentioned? 

Could it be generalised to advise that any PHP-level deprecated functions should be avoided (PHPCS warning or error)?